src/input/types.js

```diff
@@ -65,7 +65,7 @@ export type BaseInputPropsT = {
   autoFocus: boolean,
   disabled: boolean,
   error: boolean | React.Node | ((props: PropsT) => React.Node),
-  id: string,
+  id: ?string,
```

Comment:
Maybe keep it not nullable?

Reply:
This change is no longer needed. Removed.